packages/instrumenter/src/mutation-level/mutation-level.ts

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs';
+
+import {
+  ArithmeticOperator,
+  ArrayDeclaration,
+  ArrowFunction,
+  AssignmentOperator,
+  BlockStatement,
+  BooleanLiteral,
+  ConditionalExpression,
+  EqualityOperator,
+  LogicalOperator,
+  MethodExpression,
+  MutatorDefinition,
+  ObjectLiteral,
+  OptionalChaining,
+  Regex,
+  StringLiteral,
+  UnaryOperator,
+  UpdateOperator,
+} from '@stryker-mutator/api/core';
+
+export type NodeMutatorConfiguration<T> = Record<string, ReplacementConfiguration<T>>;
+
+interface ReplacementConfiguration<T> {
+  replacement?: any;
```

Comment:
I'm not sure if an `any` is needed here. I think it was strongly typed before.

Reply:
Updated the type to `Node | Node[] | boolean | string | null`